tests/ser_de.rs

```diff
@@ -97,6 +99,25 @@ fn get_test_tuples() -> Vec<(String, SignedRawInvoice, Option<SemanticError>)> {
 					)
 				}).unwrap(),
 			None
+		),
+		(
+			"lnbc20m1pvjluezpp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdq5vdhkven9v5sxyetpdeessp59g4z52329g4z52329g4z52329g4z52329g4z52329g4z52329g4q9gkzyrw8zhfxmrcxsx7hj40yejq6lkvn75l9yjmapjv94haz8x8jy2tvmgex8rnyqkj825csd2t64fu0p4ctad2cf4tgy5gh2fns6ygp6pnc3y".to_owned(),
```

Comment:
Does it make sense to include the test case from BOLT 11? `lnbc25m1pvjluezpp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdq5vdhkven9v5sxyetpdeessp5zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zygs9q5sqqqqqqqqqqqqqqqpqsq67gye39hfg3zd8rgc80k32tvy9xk2xunwm5lzexnvpx6fd77en8qaq424dxgt56cag2dpt359k3ssyhetktkpqh24jqnjyw6uqd08sgptq44qu`

Reply:
Yeah, actually you are right. @valentinewallace didn't want to use it because it uses features too (that this crate doesn't implement yet), which sounded reasonable. But for testing parsing that's a non-problem, we just treat the features field as unknown. So yes, I'd actually prefer the one out of the standard to the hand-crafted test case. Good catch.

Reply:
Ok, I added a unit test in `de.rs` using the BOLT 11 test case. I kept the other test to also have serialization testing. I'm not sure if this is the testing you expected so lmk if it's off!